tests/parser/test_osmparser.py

```diff
@@ -1,18 +1,185 @@
 import logging
 import os.path
+import tempfile
 import unittest
+import xml.etree.ElementTree as ElementTree
 
 from lafourche.parser import OsmParser
 
 
 class TestOsmParser(unittest.TestCase):
     logging.basicConfig(level=logging.DEBUG)
 
+
+    def test_get_weight_for_way_highway_pedestrian(self):
+        parser = OsmParser()
+
+        xml = """
+        <way id="819897361" visible="true" version="2" changeset="93977993" timestamp="2020-11-12T08:24:25Z" user="Chivos" uid="963055">
+          <nd ref="27323728"/>
+          <nd ref="6121940144"/>
+          <nd ref="5715654115"/>
+          <tag k="busway:left" v="opposite_lane"/>
+          <tag k="highway" v="primary"/>
+          <tag k="lit" v="yes"/>
+          <tag k="maxspeed" v="30"/>
+         </way>
+        """
+        tmp = tempfile.NamedTemporaryFile()
+        with open(tmp.name, 'w') as f:
+            f.write(xml)
+        way = ElementTree.parse(tmp)
```

Comment:
It seems that we're doing this logic of writing the string to a temp file everywhere. Would be better to refactor that into it's own function and call that function when needed